spec/plurimath/mathml/parse_spec.rb

```diff
@@ -0,0 +1,82 @@
+require_relative "../../../lib/plurimath/math"
+require_relative "../../../lib/plurimath/mathml/parse"
+require_relative "../../../lib/plurimath/mathml/constants"
+
+RSpec.describe Plurimath::Mathml::Parse do
+
+  subject(:formula) { Plurimath::Mathml::Parse.new.parse(exp.gsub("\n", "").gsub(" ", "")) }
+
+  context "contains opening and closing tag" do
+    let (:exp) { "<math xmlns='http://www.w3.org/1998/Math/MathML'></math>" }
+
+    it "matches open and close tag" do
+      expect(formula[:open]).to eq("math")
+      expect(formula[:close]).to eq("math")
+    end
+
+    it "matches attributes" do
+      expect(formula[:attributes].first[:name]).to eq("xmlns")
+      expect(formula[:attributes].last[:value]).to eq("http://www.w3.org/1998/Math/MathML")
```

Comment:
I think both should be consistent. Currently, it feels like the name and value are for different attributes.

```suggestion
      expect(formula[:attributes].first[:name]).to eq("xmlns")
      expect(formula[:attributes].first[:value]).to eq("http://www.w3.org/1998/Math/MathML")
```